### DISCUSSION TITLE
Plot size k31

### DISCUSSION THREAD

**Discussion by romeo88vn** [madMAx43v3r/chia-plotter#892] [Ideas]
Can you make another version or integrate it into the next version for plot size k31?
Thanks!

**Comment by altendky** [comment]
k31 is both too small to be valid for mainnet and still kind of big for testing.  Why do you want k31 support?

**Reply by Sunix99** [reply]
Because some fork support k31.

**Comment by vladaepro** [comment]
I'm also interested in this - or even better, even smaller plot sizes, as i'm trying to make some modifications on the code, but can't put more than 64GB RAM to my Hackintosh (development machine), and also have to wait for a long time for tests on k32

**Comment by rattfieldnz** [comment]
I'd like to have k29 enabled so MadMax would work with Chives. Where abouts in the code determines k sizes, and is there any extra logic needed to add k29 plot size?

**Reply by altendky** [reply]
Search around for k33.  At the very least, https://github.com/madMAx43v3r/chia-plotter/issues/25#issuecomment-857672918 applies.

Does Chives _only_ work with k29?  They ban k32?

**Reply by Sunix99** [reply]
Chives only works with k29, k30 and k31. Not supporting K32 and + is the main purpose of this fork.

**Comment by madMAx43v3r** [comment]
It's done, see `-k` and `-x`.
